### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by kausha15**
Added Spanish Translation.

### CODE REVIEW

File: src/locales/es.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "English": "Inglés",
+  "German": "Alemán",
+  "Submit": "Enviar",
+  "Change Language": "Cambiar idioma",
+  "Search your device": "Busca tu dispositivo",
+  "No results": "No hay resultados",
+  "Model": "Modelo",
+  "Vendor": "Vendedor",
```
Comment (gdelacruzfdez) [inline src/locales/es.json]:
```suggestion
  "Vendor": "Fabricante",
```

File: src/locales/es.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "Basic": "Básico",
+  "Advanced": "Avanzado",
+  "Build": "Construir",
+  "Cancel": "Cancelar",
+  "Add package(s)": "Agregar paquete (s)",
+  "Edit UCI defaults": "Editar valores predeterminados de UCI",
+  "Version": "Versión",
+  "Please confirm that you want to perform this action": "Confirme que desea realizar esta acción.",
+  "Building image requires computation resources, so we would request you to check if this selection is what you want": "La creación de imágenes requiere recursos de cálculo, por lo que le solicitamos que verifique si esta selección es lo que desea",
+  "warning432": "Los dispositivos con ≤4MB Flash y / o ≤32MB RAM están funcionando, pero son muy limitados (generalmente no pueden instalar o ejecutar paquetes adicionales) porque tienen poca RAM y memoria flash. Tenga esto en cuenta al elegir un dispositivo para comprar o al instalar OpenWrt en su dispositivo porque está listado como compatible. No se pueden instalar ni ejecutar paquetes adicionales porque tienen poca RAM y poca memoria flash. Tenga esto en cuenta al elegir un dispositivo para comprar, o cuando elija flashear OpenWrt en su dispositivo porque está listado como compatible.",
```
Comment (gdelacruzfdez) [inline src/locales/es.json]:
```suggestion
  "warning432": "Los dispositivos con ≤4MB Flash y / o ≤32MB RAM funcionan pero están limitados (generalmente no pueden instalar o ejecutar paquetes adicionales) porque tienen poca RAM y memoria flash. Tenga esto en cuenta al elegir qué dispositivo comprar o al instalar OpenWrt en su dispositivo porque está listado como compatible.",
```

File: src/locales/es.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "OpenWrt Firmware Selector": "Selector de firmware OpenWrt",
+  "Download OpenWrt firmware for your device!": "¡Descargue el firmware OpenWrt para su dispositivo!",
+  "Target": "Objetivo",
+  "Dismiss": "Descartar",
+  "There is an error with the packages you selected": "Hay un error con los paquetes que seleccionó",
```
Comment (gdelacruzfdez) [inline src/locales/es.json]:
```suggestion
  "There is an error with the packages you selected": "Hay un error con los paquetes seleccionados",
```

File: src/locales/es.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "Downloads": "Descargas",
+  "Please use the input below to download firmware for your device!": "¡Utilice la entrada a continuación para descargar el firmware de su dispositivo!"
```
Comment (gdelacruzfdez) [inline src/locales/es.json]:
```suggestion
  "Please use the input below to download firmware for your device!": "¡Utilice el siguiente campo para descargar el firmware de su dispositivo!"
```

**Comment by sudhanshu16** [comment]
@kausha15  @gdelacruzfdez , Thank you guys for the contributions 😄 
Closes #20

**Pull request merged**